app/validators/status_length_validator.rb

```diff
@@ -15,7 +15,7 @@ def too_long?(status)
   end
 
   def countable_length(status)
-    total_text(status).mb_chars.grapheme_length
+    total_text(status).scan(Account::GRAPHEME_RE).length
```

Comment:
Maybe we could add a spec to make sure that this works?

Reply:
Have hopefully added one